src/runtime/nitro-plugin.ts

```diff
@@ -15,5 +15,21 @@ export default <NitroAppPlugin> function (nitroApp) {
     if (i !== -1) {
       htmlContext.head[i] = htmlContext.head[i].replace(JS_HINT_RE, '')
     }
+
+    // clean up useHead artifacts
+    htmlContext.htmlAttrs = htmlContext.htmlAttrs.filter(a => !a.includes('data-head-attrs'))
+    htmlContext.bodyAttrs = htmlContext.bodyAttrs.filter(a => !a.includes('data-head-attrs'))
+    for (const k in htmlContext.head) {
+      // remove preload of api, i.e from nuxt/content
+      htmlContext.head[k] = htmlContext.head[k].replace(
+        /<link[^>]+rel="prefetch"[^>]+href="\/api\/[^"]*"[^>]*>/g,
```

Comment:
I was trying to find some config for the `/api` nitro prefix but couldn't find it, is it static?